matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/pushrules/rest/PushRule.kt

```diff
@@ -100,6 +100,13 @@ data class PushRule(
         )
     }
 
+    /**
+     * Get the highlight status. As spec mentions assume false if no tweak present.
+     */
+    fun getHighlight(): Boolean {
+        return (getActions().firstOrNull { it is Action.Highlight } as? Action.Highlight)?.highlight ?: false
```

Comment:
You can rewrite to 
`return getActions().filterIsInstance<Action.Highlight>().firstOrNull()?.highlight.orFalse()`